lsp/nls/src/config.rs

```diff
@@ -0,0 +1,72 @@
+//! Configuration for the nls server
```

Comment:
nitpick: NLS already means Nickel language server, so NLS server sounds odd

```suggestion
//! Configuration for Nickel Language Server
```

Reply:
```suggestion
//! Configuration for the Nickel nls Language server
```

? :smile: